tools/logs-analyzer/github.json

```diff
@@ -0,0 +1,4618 @@
+{
```

Comment:
Just out of curiosity, what is this file? Do you know? It does seem to contain sensitive information...

Reply:
those are the public ip cidr blocks provided by Github, Google, etc. to perform the network analysis (was looking at safelisting any network call to those known ranges